common/djangoapps/xblock_django/models.py

```diff
@@ -21,6 +23,15 @@ class Meta(ConfigurationModel.Meta):
         help_text=_('Space-separated list of XBlocks which should not render.')
     )
 
+    deprecated_blocks = TextField(
```

Comment:
Can we call this something else like `disabled_create_blocks`? We already have a setting named `DEPRECATED_BLOCK_TYPES` which does something different.